### PR TITLE
starts round three

### PR DESCRIPTION
**Pull request by fpigeonjr**
## What changed

- refactors based on @stevtek and @kimschulke feedback
- updates Services Component state to match backend
- adds Services Component name to delete modal

## Issue

#1852 

## How to test

_Write out steps for how someone could test this PR against the acceptance criteria_

## Screenshots

_If relevant, e.g. for a front-end feature_

## Definition of Done Checklist
- [X] OESA: Code refactored for clarity
- [-] OESA: Dependency rules followed
- [X] Automated unit tests updated and passed
- [X] Automated integration tests updated and passed
- [X] Automated quality tests updated and passed
- [X] Automated load tests updated and passed
- [X] Automated a11y tests updated and passed
- [X] Automated security tests updated and passed
- [X] 90%+ Code coverage achieved
- [-] Form validations updated

### CODE REVIEW

File: frontend/src/pages/servicesComponents/servicesComponents.constants.js [hunk 1]
```diff
@@ -12,6 +12,22 @@ export const initialServicesComponent = {
         "Develop a theory of change and identify ways to improve the program through continuous user feedback and engagement"
 };
 
+export const backendServicesComponents = {
```
Comment (fpigeonjr) [inline frontend/src/pages/servicesComponents/servicesComponents.constants.js:15]: am I doing it right @stevtek?

File: frontend/src/pages/servicesComponents/servicesComponents.constants.js [hunk 2]
```diff
@@ -1,15 +1,33 @@
...
-    popEndDay: "15",
-    popEndYear: "2025",
-    description:
-        "Develop a theory of change and identify ways to improve the program through continuous user feedback and engagement"
+export const SERVICE_REQ_TYPES = {
+    NON_SEVERABLE: "Non-Severable",
+    SEVERABLE: "Severable"
+};
+
+export const NON_SEVERABLE_OPTIONS = ["SC1", "SC2", "SC3", "SC4", "SC5", "SC6"];
```
Comment (tdonaworth) [inline frontend/src/pages/servicesComponents/servicesComponents.constants.js:6]: The idea was to have these be data driven; the API call should return the name needed (without needing translated).

Comment (stevtek) [reply frontend/src/pages/servicesComponents/servicesComponents.constants.js:6]: I think the challenge with the UI currently is we need to display something before it's created.

Comment (fpigeonjr) [reply frontend/src/pages/servicesComponents/servicesComponents.constants.js:6]: like Steve said, these would be used to build the dropdowns on the FE based on the index. I imagine once we get the api calls in place further refinement will be needed.

**Comment by fpigeonjr** [comment]
PR #1965 and #1968 build off of this but make substantial changes, closing PR

**Pull request closed without merging**